source/processes/ns_hpv.cpp

```diff
@@ -208,6 +209,23 @@ struct NsService : SessionToPort {
             auto description = fmt::format( "LaunchTitle, title_id={:#x}, flags={:#x}", title_id, flags);
             Session::OnRequest(hypervisor, thread, session, description);
         });
+        
+        dispatcher.DecodeRequest<NS::CardUpdateInitialize>([&](auto& response, uint32_t shared_mem_size, Handle /* shared_mem */) {
+            auto description = fmt::format( "CardUpdateInitialize, shared_mem_size={:#x}", shared_mem_size);
+            Session::OnRequest(hypervisor, thread, session, description);
+            
+            auto gamecard = thread.GetOS().setup.gamecard.get();
+            assert(gamecard);
+            // Disable card update process for 3DSX files, CXI files, and CCI images without an update partition
```

Comment:
Since HPV is normally used for introspection only, let's be explicit about what's happening here:
```suggestion
            // Override the request data to disable card update process for 3DSX files, CXI files, and CCI images without an update partition
```